### PR TITLE
nebula: init at 1.0.0

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/73918)

### CODE REVIEW

**Comment by erikarvstedt** [comment]
Tested ACK 0de99691d42f9890dced7aa757356096dceca637

File: pkgs/tools/networking/nebula/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "slackhq";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0j7fna352z8kzx6n0hck7rp122c0v44j9syz0v30vq47xq2pwj5c";
+  };
+
+  modSha256 = "130h0gc5z0w5inpc99y6mismwg3nyzk3bqdq5v9yclkxlhkbcp6d";
```
Comment (coderobe) [inline pkgs/tools/networking/nebula/default.nix]: This `modSha256` does not match up with what i'm getting

**Comment by kalbasit** [comment]
@GrahamcOfBorg build nebula

File: pkgs/tools/networking/nebula/default.nix (hunk 1 above)
Comment (kalbasit) [reply pkgs/tools/networking/nebula/default.nix]: build passed on ofborg.

**Review by kalbasit — APPROVED** [review]

**Comment by devhell** [comment]
Would this ideally benefit from a service as well?

**Comment by coderobe** [comment]
upstream is working on that, better just wait on the next upstream release.

**Comment by filalex77** [comment]
@devhell Just as @coderobe said: as soon as they get the service done, I'll publish my draft of the service.

**Pull request merged**